### DISCUSSION TITLE
HM-Sen-MDIR-O-3 light sensor does not have "lux" unit

### DISCUSSION THREAD

**Discussion by ps915** [SukramJ/aiohomematic#1743] [General]
I just have installed twi HM-Sen-MDIR-O-3 and on both devices. On both devices, i dont get a "lumen" unit on the light sensor.
Is this normal?

<img width="1247" alt="Bildschirmfoto 2024-10-04 um 09 44 39" src="https://github.com/user-attachments/assets/cfe7d52e-ecb5-44e2-979b-c75ddf47eb5d">

**Comment by SukramJ** [comment]
Lux ist die Einheit der Beleuchtungsstärke und nicht der Helligkeit.

**Comment by ps915** [comment]
Okay, dann ist es Lumen?! Jedenfalls wird keine der beiden Einheiten angezeigt! ;)

**Reply by SukramJ** [reply]
> Okay, dann ist es Lumen?! Jedenfalls wird keine der beiden Einheiten angezeigt! ;)

Lumen ist der Lichtstrom. Ich würde an deiner Stelle da mal genauer recherchieren.
In HA hat die Helligkeit keine Einheit und auch die eq-3 Geräte liefern nichts mit.

Wenn du unbedingt eine Einheit möchtest kannst Du dir an die Entität per customize selber was dran konfigurieren.

**Comment by Baxxy13** [comment]
Die HM-Geräte liefern doch nur einen einheitslosen Wert von 0 - 255. 
Erst die HmIP-Geräte liefern da was "passenderes*.
